MdePkg/Library/BaseIoLibIntrinsic/X64/IoFifoSev.nasm

```diff
@@ -0,0 +1,282 @@
+;------------------------------------------------------------------------------

```

Comment:
This file is not required. It is for AMD SEV implementation.

Reply:
Let me remove this. Thanks.